### PR TITLE
Use temba-datepicker instead of lit-flatpickr

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

**Comment by ericnewcomer** [comment]
@rowanseymour still might be worth pushing this if we do it early enough tomorrow

File: temba/msgs/views.py [hunk 1]
```diff
@@ -653,6 +646,18 @@ class Export(ModalMixin, OrgPermsMixin, SmartFormView):
         submit_button_name = "Export"
         success_url = "@msgs.msg_inbox"
 
+        def derive_initial(self):
+            initial = super().derive_initial()
+
+            # default to last 90 days in org timezone
+            tz = self.request.user.get_org().timezone
```
Comment (rowanseymour) [inline temba/msgs/views.py:653]: `self.request.org`

File: temba/msgs/views.py [hunk 2]
```diff
@@ -653,6 +646,18 @@ class Export(ModalMixin, OrgPermsMixin, SmartFormView):
...
+            end = datetime.datetime.now(tz)
+            start = end - timedelta(days=90)
+
+            initial["end_date"] = end.strftime("%Y-%m-%d")
+            initial["start_date"] = start.strftime("%Y-%m-%d")
```
Comment (rowanseymour) [inline temba/msgs/views.py:658]: Shouldn't these work as date objects? e.g. `start.date()`

File: temba/msgs/views.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import datetime
```
Comment (rowanseymour) [inline temba/msgs/views.py:1]: our sorta convention is `from datetime import datetime`

File: temba/msgs/views.py (hunk 1 above)
Comment (rowanseymour) [inline temba/msgs/views.py:653]: `self.request.org.timezone`

File: temba/utils/fields.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import datetime
```
Comment (rowanseymour) [inline temba/utils/fields.py:1]: `from datetime import datetime`

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/msgs/views.py (hunk 2 above)
Comment (ericnewcomer) [reply temba/msgs/views.py:658]: That would return a zeroed out datetime, ie, `2020-09-16 00:00:00`. I didn't look for another way, but strftime here seems fine to me.

Comment (ericnewcomer) [reply temba/msgs/views.py:658]: Oh, actually this works..

`end.date().isoformat()`

Comment (rowanseymour) [reply temba/msgs/views.py]: But why a string? Shouldn't it work with a date value? Like a regular Django `DateField` would let you do `DateField(initial=date(2022, 1, 1))`

Comment (rowanseymour) [reply temba/msgs/views.py]: Like if you have a model form and a date field on a model, this field needs to understand date values.. obviously at the widget level that becomes an ISO formatted string.

Comment (ericnewcomer) [reply temba/msgs/views.py]: Can make that change, would be in the component, so needs a new release..

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**